spec/lib/lift_wing_api_spec.rb

```diff
@@ -70,15 +75,24 @@
     it 'handles timeout errors' do
       stub_request(:any, 'https://api.wikimedia.org')
         .to_raise(Errno::ETIMEDOUT)
-      expect_any_instance_of(described_class).to receive(:log_error).once
+      expect(lift_wing_api_class_en_wiki).to receive(:log_error).once
```

Comment:
Changed this to avoid using `expect_any_instance_of` here, as the spec then makes statements on `subject0`, which is a particular instance. I don't like making expectations to any instance and to a particular instance in the same spec, I think it's confusing.